src/pages/LendPage/LendPage.tsx

```diff
@@ -0,0 +1,251 @@
+import React, { useEffect, useState } from 'react';
+import Web3 from 'web3';
+import { Box } from '@material-ui/core';
+import { Skeleton } from '@material-ui/lab';
+import { SearchInput, CustomMenu, CurrencyLogo } from 'components';
+import { useHistory } from 'react-router-dom';
+import { useActiveWeb3React } from 'hooks';
+import { MarketSDK, Comptroller, Pool, PoolDirectoryV1 } from 'market-sdk';
+import { midUsdFormatter } from 'utils/bigUtils';
+import {
+  fetchPoolData,
+  USDPricedPoolAsset,
+} from 'utils/marketxyz/fetchPoolData';
+import { useTranslation } from 'react-i18next';
+import 'pages/styles/lend.scss';
+import { getPoolAssetToken } from 'utils/marketxyz';
+import { Token } from '@uniswap/sdk';
+
+const QS_PoolDirectory = '0x9180296118C8Deb7c5547eF5c1E798DC0405f350';
+const QS_Pools = ['0x772EdfEDee10029E98AF15359595bB398950416B'];

```

Comment:
Shouldn't this be pulled from web 3 why is this hardcoded? where is the admin page